src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2983,12 +2983,14 @@ namespace UnitarySystems {
         if (this->m_CoolCoilExists && this->m_MaxCoolAirVolFlow < 0.0) {
             if (!state.dataSize->SysSizingRunDone) {
                 int BranchNum = BranchInputManager::GetAirBranchIndex(state, "AirloopHVAC:UnitarySystem", this->Name);
-                std::string FanType = "";
-                std::string m_FanName = ""; // the notation m_ implies member variables, and this is a local
                 BranchFanFlow = 0.0;
-                if (BranchNum > 0.0) BranchInputManager::GetBranchFanTypeName(state, BranchNum, FanType, m_FanName, ErrFound);
-                if (!ErrFound && BranchNum > 0) {
-                    BranchFanFlow = state.dataFans->fans(this->m_FanIndex)->maxAirFlowRate;
+                if (BranchNum > 0.0) {
+                    std::string FanType = "";
+                    std::string FanName = "";
+                    BranchInputManager::GetBranchFanTypeName(state, BranchNum, FanType, FanName, ErrFound);
+                    if (!ErrFound) {
+                        BranchFanFlow = state.dataFans->fans(this->m_FanIndex)->maxAirFlowRate;
+                    }
```

Comment:
[src/EnergyPlus/UnitarySystem.cc:2986]:(style),[variableScope],The scope of the variable 'FanType' can be reduced.
[src/EnergyPlus/UnitarySystem.cc:2987]:(style),[variableScope],The scope of the variable 'm_FanName' can be reduced.